<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FmExtractOriginal</class>
 <widget class="QDialog" name="FmExtractOriginal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>239</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Extract original</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="grpChannel">
     <property name="title">
      <string>Texts</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="QRadioButton" name="radioChanOriginal">
        <property name="text">
         <string>Original</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioChanTranslation">
        <property name="text">
         <string>Translation</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QCheckBox" name="chkUseOriginal">
          <property name="text">
           <string>Use original if untranslated</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpComments">
     <property name="title">
      <string>Comments</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="QRadioButton" name="radioCommAu">
        <property name="text">
         <string>Author’s</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioCommAuTr">
        <property name="text">
         <string>Author’s → translator’s</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioCommTr">
        <property name="text">
         <string>Translator’s</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioCommTrAu">
        <property name="text">
         <string>Translator’s → author’s</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
